src/main/java/net/coderbot/iris/shaderpack/IdMap.java

```diff
@@ -70,10 +78,15 @@
 
 		// TODO: Properly override block render layers
 
-		if (blockPropertiesMap == null) {
-			// Fill in with default values...
+		// Fill in with default values...
+		if (blockPropertiesMap == null || blockPropertiesMap.isEmpty()) {
```

Comment:
This is invalid - if a shader pack specifies an empty block.properties file, that means that all blocks map to `-1`. Only the absence of a block properties file should be interpreted as the shader pack using legacy mappings.